src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklySettings.kt

```diff
@@ -1,18 +1,25 @@
 package com.launchdarkly.intellij.settings
 
+/* ktlint-disable import-ordering */
```

Comment:
Even manually re-adding the imports 1 by 1 ended up with a sorting issue. We can come back and revisit this disable.